src/Sylius/Bundle/ApiBundle/Controller/OrderController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Sylius\Bundle\ApiBundle\Command\Cart\RemoveItemFromCart;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class OrderController
+{
+    /** @var MessageBusInterface */
+    private $commandBus;
+
+    public function __construct(MessageBusInterface $commandBus)
+    {
+        $this->commandBus = $commandBus;
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $command = new RemoveItemFromCart();
```

Comment:
```suggestion
        $command = new RemoveItemFromCart($request->attributes->get('id'), $request->attributes->get('itemId'));
```
wouldnt it be better?